tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/MatterProvision.cs

```diff
@@ -1147,35 +1177,33 @@ private static string TeamMembersPermissionInformation(MatterDetails matterDetai
 
             List<string> keys = new List<string>();
             Dictionary<string, string> propertyList = new Dictionary<string, string>();
-            keys.Add(matterSettings.StampedPropertyPracticeGroup);

```

Comment:
Now all the stamped properties will be read with in a for loop. Tomorrow if any other stamped property has been added, no need to make any changes to the code